angular/package.json

```diff
@@ -41,8 +41,8 @@
     "lint": "npm-run-all -l -s -c lint:ts:* lint:html lint:scss",
     "lint:ts": "npm-run-all -l -s -c lint:ts:*",
     "lint:ts:1": "ng lint --format=stylish",
-    "lint:ts:2": "prettier projects/**/*.ts --warn",
-    "lint:html": "prettier projects/**/*.html --warn",
+    "lint:ts:2": "prettier projects/**/*.ts --check",
```

Comment:
Lets not enable this right now.

Reply:
Enable it? It runs the same before and after. The only difference is that with `--check` it wont print the entire contents of files that fail.  Instead it will only print the name of the file.  Just makes it easier to read.

Reply:
We ran into many issues with the environment vs. local, and it's separate from the bug that this task is supposed to be addressing.  We can file a separate PR instead.